components/atom/switch/src/SwitchType/single.js

```diff
@@ -65,6 +65,7 @@ export const SingleSwitchTypeRender = forwardRef(
               name={name}
               text={defaultLabelLeft ? label : labelLeft}
               optionalText={labelOptionalText}
+              inline={fullWidth ? 'left' : 'right'}
```

Comment:
@jordevo I think this is your proposal but it doesn't work

Reply:
@jordevo The inline prop should change the text-align but it changes the margin. It can't work :(